prime-router/src/main/kotlin/fhirengine/engine/FHIRConverter.kt

```diff
@@ -137,68 +140,92 @@ class FHIRConverter(
                     )
 
                     maybeParallelize(
-                        fhirBundles.size,
-                        Streams.mapWithIndex(fhirBundles.stream()) { bundle, index ->
+                        processedItems.size,
+                        Streams.mapWithIndex(processedItems.stream()) { bundle, index ->
```

Comment:
Why `Streams.mapWithIndex(processedItems.stream())` vs `processedItems.mapIndexed`?

Reply:
We want to be able to do this logic in parallel if there are enough items which is only available via the Java streams API

Reply:
Ah, I didn't know that wasn't available with the kotlin version of that. Boo